<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShrewDriver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QSpinBox" name="spSustain">
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbSustain">
      <property name="text">
       <string>Stim Sustain  (ms)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="spRampDown">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spRampUp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cbWaveform"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbWaveform">
      <property name="text">
       <string>Waveform</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbPower">
      <property name="text">
       <string>Max Power (mW/mm^2)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lbRampUp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ramp Up (ms)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lbRampDown">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ramp Down (ms)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <layout class="QVBoxLayout" name="wgMPL"/>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="cbxCycles">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Pulse Train (cycles)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="spCycles">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="spISI">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="lbISI">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>ISI (ms)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="btnOpto">
      <property name="text">
       <string>Opto LED</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="btnDummy">
      <property name="text">
       <string>Dummy LED</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cbWindow"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbWindow">
      <property name="text">
       <string>Window Radius (mm)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spPower"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
